classes/models/feedback.php

```diff
@@ -278,19 +278,19 @@ public function get_coursemodule_id() {
      * @todo get rid of this.
      *
      * @static
-     * @param $submission
+     * @param submission $submission
```

Comment:
is that a param type?

Reply:
I guess function is declared as (submission $submission,
                                                $isfinalgrade = 0,
                                                $assessorid = 0 )

So that's where it got it from

Reply:
Yes it's a param type for mod_coursework\models\submission